ui/lib/replication/addon/routes/application.js

```diff
@@ -14,6 +13,23 @@ export default Route.extend(ClusterRoute, {
   store: service(),
   auth: service(),
   router: service(),
+  capabilities: service(),
```

Comment:
I thought about not using the new service here so that it more cleanly backported to older versions (this only exists in 1.18 so far), BUT the service is nice because we only make **one** request for all of these capabilities instead of a request for each path which I think is a win! 🙌